.commitlintrc.mjs

```diff
@@ -24,7 +24,8 @@ export default {
         'exporter-js',
         'exporter-scss',
         'exporter-svg',
-        'exporter-variables-scss',
+        'spirit-assets-exporter',
```

Comment:
I prefer `exporter-tokens` and `exporter-assets`. I think we do not need to use the `spirit` prefix here. So the names remained short and sorted based on context (`exporter-`).